test/test-patientsController.js

```diff
@@ -424,7 +424,7 @@ describe('patientsController', () => {
         .set('Cookie', `token=${cookie.getJwtTokenForUser(psy.dossierNumber, 'randomXSRFToken')}`)
         .set('xsrf-token', 'randomXSRFToken')
         .then(async (res) => {
-          res.body.success.should.equal(false);
+          res.status.should.equal(400);
```

Comment:
I think we should return same http status as previous test. The user will therefore not know whether the patient does not exists or is not his/hers. I would put 404 here.

Reply:
this is a bit different, the uuid validation is done by the type validator => 400